Source/ASDisplayNode.mm

```diff
@@ -453,6 +453,11 @@ - (void)asyncTraitCollectionDidChangeWithPreviousTraitCollection:(ASPrimitiveTra
           CGFloat cornerRadius = _cornerRadius;
           ASCornerRoundingType cornerRoundingType = _cornerRoundingType;
           UIColor *backgroundColor = _backgroundColor;
+          if (@available(iOS 13.0, *)) {
+             UITraitCollection *tempTraitCollection = [UITraitCollection traitCollectionWithUserInterfaceStyle:_primitiveTraitCollection.userInterfaceStyle];
```

Comment:
in general this is a valid change, but we found that in some cases we have a discrepancy between texture and system `userInterfaceStyle`. E.g. when you background the app, it changes  `userInterfaceStyle` back and forth for snapshotting.